feature-configs/5g-ref-nw/clusters/otwaon1234rd/cu-up/README.md

```diff
@@ -0,0 +1,10 @@
+# Data center CU-UP
+This is an example of a data center CU-UP configurations
+## Structure
+The structure is identical to [du-large-m1](../otwaon1234rd/du-large-m1).
+
+## Labeling
+In addition to having the master role, the node must be labeled as follows:
+- `node-role.kubernetes.io/worker-cu-up-otwaon1234rd=""` - used for machine config pool
+- `ptp/slave=""` - used as node selector for PTP
```

Comment:
I do not think we would enable PTP in the general CU example.  I do think we would enable NTP.  Should we add an NTP provisioning via MCO?

Reply:
fixed